doc/masterminter.md

```diff
@@ -0,0 +1,101 @@
+# MasterMinter contract
+The MasterMinter is a governance contract.  It delegates the functionality
+of the `masterMinter` role in the Centre USDC contract to multiple addresses.
```

Comment:
I think we always use the upper case of CENTRE.

Reply:
fixed